### DISCUSSION TITLE
Enhance scoop user experience for shell users in Windows -- WSL/Git-Bash users

### DISCUSSION THREAD

**Discussion by edwinjhlee** [ScoopInstaller/Scoop#5865] [Show and tell]
Dear friends of scoop community,

I am developing [x-cmd](https://x-cmd.com) in recent 3 years. 

I've been working on a way to make Scoop easier to use for people who work in WSL (Windows Subsystem for Linux) or Git Bash. 
I also add an interactive to search scoop softwares. Below is the screenshot, and welcome to our [demo page](https://www.x-cmd.com/mod/scoop).

<img width="857" alt="image" src="https://github.com/ScoopInstaller/Scoop/assets/4426319/f8ee75ad-6908-44d9-ad84-d472864f504e">

Also, in the scoop module, we provide a convinient way to install scoop. Windows user only need to type 'x scoop install', or just 'x scoop', x-cmd will help download and install scoop automatically.

I was once a choco user in 2016, and has become a mac user since 2018. However, I still am happy to see there is an opensource and powerful package management tool in windows, which make windows much more attractive for terminal users.

Regards,
Li Juanhao

**Comment by rashil2000** [comment]
Scoop can readily be used inside WSL or Git Bash/MSYS2 environments, since we provide .sh scripts in addition to the .cmd and .ps1 scripts.
